src/utils/rojo-lock-manager.ts

```
@@ -0,0 +1,126 @@
+
+	if (Number.isNaN(pid) || Number.isNaN(port) || Number.isNaN(startTime)) {
+		// Invalid lockfile, clean it up
+		await cleanupLockfile(lockPath);
+		return null;
+	}
+
+	// Check if process is still running
+	const isRunning = await isProcessAlive(pid);
+	if (!isRunning) {
+		// Stale lockfile, clean it up
+		log.warn(`Cleaned up stale Rojo lockfile (PID ${pid} no longer running)`);
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)